models/generators.py

```diff
@@ -71,48 +95,59 @@ def __init__(self, img_size, num_latent_dims, kernel_size, num_filters,
                     Dimensionality of latent input.
                 kernel_size : tuple
                     (height, width)
-                reshape_dims : tuple or list
+                reshape_dims : tuple or list TODO: actually use
                     [91, 125, 128]
                 lr : float
         """
-        super().__init__()
-
-        self.w_init = tf.random_normal_initializer(stddev=0.02)
-        self.bn_init = tf.random_normal_initializer(1., 0.02)
-
-        self.img_size = img_size
-
+        super().__init__(img_size, num_latent_dims, lr, conditional_emb_size)
         num_output_channels = self.img_size[0]
```

Comment:
you sure img_size is (3, 64, 64) and not the normal convention which is (64, 64, 3)?

Reply:
I'm not. It appeared that way from the existing code though. I'mm double check/

Reply:
Yup looks like it is so